google/cloud/firestore_v1/order.py

```diff
@@ -29,12 +29,12 @@ class TypeOrder(Enum):
     REF = 6
     GEO_POINT = 7
     ARRAY = 8
-    OBJECT = 9
+    VECTOR = 9
+    OBJECT = 10
```

Comment:
This is a breaking change. If we don't think users will be impacted, please add a comment in the PR. This change should be captured in the releases notes (regardless if it is considered breaking or not)

Reply:
Good catch. Refactor to decouple order from TypeOrder